.github/pull_request_template.md

```diff
@@ -6,6 +6,7 @@
 
 - [ ] I've made sure the lint is passing in this PR.
 - [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
+- [ ] I've checked the new test coverage and the coverage percentage didn't drop.
```

Comment:
It's nice if coverage can be at least maintained in each PR, but it may be too difficult (passing lint and unit tests on the other hand are must)